src/Greeting.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import greetingsActions from './redux/greetings/actions';
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  const greeting = useSelector((state) => state.greeting);
+  useEffect(() => dispatch(greetingsActions()), []);
+  return (
+    <div>
+      <h1>Greetings</h1>
+      <p>{greeting || 'Searching please wait...'}</p>
+    </div>
```

Comment:
- [ ] Hello make sure the dispatch that  you are calling to render the greeting messages from the Rails API it does give the random messages instead of 'Searching please wait...'.